File: spring-boot-mybatis/mybatis-mybatisplus/src/main/java/com/xphsc/mybatis/service/UserService.java
package com.xphsc.mybatis.service;



import com.xphsc.mybatis.model.response.UserDTO;

import java.util.List;

/**
 * Created by ${huipei.x} on 2017-2-25.
 */
public interface UserService {
    public List<UserDTO> listUser();
}

File: spring-boot-kotlin/src/main/java/com/xphsc/kotlin/service/UserService.kt
package com.xphsc.kotlin.service

import com.xphsc.kotlin.model.User
import com.xphsc.kotlin.model.response.UserDTO

/**
 * Created by ${huipei.x} on 2017-2-25
 */
interface UserService {
    fun listUser(): List<UserDTO>;
}